Misc/NEWS.d/next/Core and Builtins/2024-03-29-15-04-13.gh-issue-117349.OB9kQQ.rst

```diff
@@ -0,0 +1 @@
+Hardcode constants in :mod:`ntpath`.
```

Comment:
NEWS entries should aim to describe the impact the change will have for people using CPython and the stdlib, not the implementation details of how the change was achieved. In this case, the entry should probably just be something like this:

```suggestion
Optimise several functions in :mod:`os.path`.
```

You can also give yourself credit in the news entry, if you like ("Patch by XXX", replacing XXX with your name) -- but we prefer real names rather than "Nice Zombies", if possible :-)

Reply:
Also FYI, you've made enough contributions now for you to add yourself to https://github.com/python/cpython/blob/main/Misc/ACKS, if you like (but it's not mandatory)